docroot/modules/custom/dcc_gtd_registration/src/Form/GlobalTrainingRegistrationForm.php

```diff
@@ -0,0 +1,482 @@
+<?php
+
+namespace Drupal\dcc_gtd_registration\Form;
+
+use Drupal\Core\Ajax\AjaxResponse;
+use Drupal\Core\Ajax\PrependCommand;
+use Drupal\Core\Ajax\ReplaceCommand;
+use Drupal\Core\Entity\Entity;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Locale\CountryManager;
+use Drupal\Core\Render\Element\StatusMessages;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class GlobalTrainingRegistrationForm
+ *
+ * @package Drupal\dcc_gtd_registration\Form
+ */
+class GlobalTrainingRegistrationForm extends FormBase {
+
+  /**
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   *  The entity type manager.
+   */
+  protected $entityTypeManager;
+
+  /**
+   * @var \Drupal\dcc_gtd_registration\Form\CountryManager
+   *  The country manager.
+   */
+  protected $countryManager;
+
+  /**
+   * GlobalTrainingRegistrationForm constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *  The entity type manager.
+   * @param \Drupal\Core\Locale\CountryManager $countryManager
+   *  The country manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, CountryManager $countryManager) {
```

Comment:
Please use CountryManagerInterface, if you pass CountryManager the dependency injection cannot be used.